### PR TITLE
Prac 06 code review request

### PR DESCRIPTION
**Pull request by hayleeturk**
@daynemorris

### CODE REVIEW

File: guitar.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return f"{self.name} ({self.year} : ${self.cost})"
+
+    def get_age(self):
+        """Calculate age of guitar"""
+        age = CURRENT_YEAR - self.year
+        return age
+
+    def is_vintage(self, age):
+        """Determine if guitar is vintage (older than 50 years)"""
+        if age > 50:
```
Comment (daynemorris) [inline guitar.py]: Could use a CONSTANT in place of 50.

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        print(guitar_data, "added.")
+        name = input("Name: ")
+    print("These are my guitars:")
+    vintage_string = ""
+    for i, guitar in enumerate(my_guitars, 1):
+        vintage_string = determine_if_vintage(guitar)
+        print("Guitar {0}: {1.name:>20} ({1.year}), worth ${1.cost:10,.2f}{2}".format(i, guitar, vintage_string))
+
+
+def get_valid_number(prompt):
```
Comment (daynemorris) [inline prac_06/guitars.py:35]: Good use of error checking.

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    """Get a valid number from user."""
+    number = float(input(prompt))
+    while number < 1:
+        print("Invalid number")
+        number = float(input(prompt))
+    return number
+
+
+def determine_if_vintage(guitar):
```
Comment (daynemorris) [inline prac_06/guitars.py:44]: Does the function in the class (is_vintage()) already do this? Try add it to the for loop.

**Comment by daynemorris** [comment]
Overall really good job. I do believe you mixed up programming_language and languages as in the file names. Just tweak that and a few best practice items and you should be good.

File: prac_06/guitars.py (hunk 3 above)
Comment (hayleeturk) [reply prac_06/guitars.py:44]: I'm unsure what you mean by this. The "is_vintage" function only checks if it is greater than the vintage age. Do you mean the for loop that enumerates through the guitars list? Can you please explain a little more? Thanks :))

Comment (lindsaymarkward) [reply prac_06/guitars.py:44]: You don't need a function just to call a method.
`guitar.is_vintage()` does the same thing as `determine_if_vintage(guitar)`

**Pull request merged**